### DISCUSSION TITLE
Is spdlog going header only only?

### DISCUSSION THREAD

**Discussion by dylanetaft** [gabime/spdlog#2959] [General]
I noticed when I compile the default git branch 1.x, I don't get a static lib.  Install does install the headers.

If I compile the last prod release, v1.12.0, I get the static .la lib when I run make.

**Comment by tt4g** [comment]
spdlog was a header-only library until a few years ago. Now you can build static library and shared library.
There is still the option to use header-only.

So if the 1.x branch refers to an older commit, it is header-only.

**Reply by dylanetaft** [reply]
Hm, it was head of the default branch.   Using the tag of the last prod release is building the library fine so I'll stick with that.  

It's possible Add_ExternalProject pulled the wrong version from git into my superbuild when I did not specify a branch or tag.  Maybe it pulled master instead of 1.x.

**Comment by gabime** [comment]
The v2.x branch however, does support header only..
